provider/lxd/storage.go

```diff
@@ -308,7 +311,7 @@ func parseFilesystemId(id string) (lxdPool, volumeName string, _ error) {
 	return fields[0], fields[1], nil
 }
 
-func destroyControllerFilesystems(env *environ, controllerUUID string) error {
+func destroyControllerFilesystems(env *environ, _ string) error {
```

Comment:
why is this correct to do?
If we are grabbing the controller UUId from config (which is what the context seems to say), shouldn't we just not take the string?

Reply:
Good catch. I blanked it because it was unused, but it looks like a prior copy/paste error. The correction passes through the controllerUUID.

I have made a note to add a test for this (when replacing the stub with mocks).